hook_scripts/pre-commit-rst-placeholder.sh

```diff
@@ -16,10 +16,9 @@ for rst in $RST_FILES; do
     bname=$(basename $rst .rst)
 
     # Check if the corresponding qmd file exists
-    if [ ! -f model/docs/${bname}.qmd ]; then
+    if [ ! -f ${TUTORIALS_DIR}/${bname}.qmd ]; then
         echo "Removing $rst"
         rm $rst
-        COUNTER=$((COUNTER+1))
```

Comment:
Why wouldn't you count this as a change for failing? So, if I understand correctly, if the file doesn't exist (b/c the tutorial was removed), then this should be changed, and pre-commit should fail, no?

Reply:
Good catch; that should not have been removed. Will revert.

Reply:
Reverted in 9127df3